README.md

```diff
@@ -32,3 +32,120 @@ dapp test -m <REGEX>
 dapp test -m testName
 dapp test -m ':ContractName\.'
 ```
+
+## Deploy to a local testnet
+Start a local testnet node and let it run in the background:
+
+```bash
+dapp testnet
+```
+
+Set up environment variables.
+See instructions for public network deployment to see all the options.
+To automatically set bare minimum environment variables run:
+
+```bash
+source scripts/local-env.sh
+```
+
+Run deployment:
+
+```bash
+scripts/deploy.sh
+```
+
+## Deploy to a public network
+
+Use dapp.tools' `ethsign` to query or add keys available on the system for signing transactions.
+
+Set up environment variables controlling the deployment process:
+
+```bash
+# The RPC URL to use, e.g. `https://mainnet.infura.io/MY_INFURA_KEY`.
+# Contracts will be deployed to whatever network that endpoint works in.
+export ETH_RPC_URL="<URL>"
+
+# Address to use for deployment. Must be available in `ethsign`
+export ETH_FROM="<ADDRESS>"
+
+# OPTIONAL
+# The file containing password to decrypt `ETH_FROM` private key from keystore.
+# If not set, the password will be prompted multiple times during deployment.
+# If `ETH_FROM` is not password protected, can be set to `/dev/null`.
+export ETH_PASSWORD="<KEYSTORE_PASSWORD>"
+
+# OPTIONAL
+# The API key to use to submit contracts' code to Etherscan.
+# In case of deployments to networks other than Ethereum an appropriate equivalent service is used.
+export ETHERSCAN_API_KEY="<KEY>"
+
+# OPTIONAL
+# The JSON file to write deployment addresses to. Default is `./deployment_<BLOCKCHAIN_NAME>.json`.
+export DEPLOYMENT_JSON="<PATH>"
+```
+
+Set up environment variables configuring the deployed contracts:
+
+```bash
+
+# OPTIONAL
+# Address of Reserve to use. If not set, a new instance is deployed.
+export RESERVE="<ADDRESS>"
+
+# OPTIONAL
+# Address of Reserve owner to set. If not set, `ETH_FROM` is used.
+export RESERVE_OWNER="<ADDRESS>"
```

Comment:
I think technically the `RESERVE_OWNER` and the `DRIPS_HUB_ADMIN` could be different addresses.

However, it might be the same in most cases.

One `GOVERNANCE` might be easier?

Reply:
That's a good point, added `GOVERNANCE`.